SU-SWS/stanford_basic
# D8CORE-1430: Fix nav color & focus state event

# READY FOR REVIEW 

# Summary
- Fix focus to only select first item when using keyboard nav.
- Fixes the home page red highlight on desktop nav when active.

# Review By (Date)
- Thursday

# Urgency
- Med/Low

# Steps to Test

1. On a fresh build of the 1.x branch of cardinalsites navigate to the home page
2. On Desktop (LG +) See that the home page highlight is red
3. Click on the `resources` menu item and on page load see that the highlight is black
4. Resize your browser window to MD or small and expand the navigation. See that the first item is focused and is highlighted.
5. Check out this branch and clear all caches
6. Navigate back to the home page and see that the menu item now has a black highlight on Desktop sized screens.
7. Resize your browser window to MD or small and expand the navigation. See that the first item is NOT focused and is NOT highlighted when using the mouse.
8. Refresh the page and use keyboard to tab to hamburger
9. Hit enter or space to expand the menu and see focus set to first item.

# Affected Projects or Products
- D8CORE

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
